src/components/SnazzyButton.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import { snazzyButtonStyle } from './helpers';
+import isSnazzy from 'src/components/SnazzyButton';
```

Comment:
Delete this line, you do not need to import the component into itself. 
This would create a bug.